docs/source/tutorials.rst

```diff
@@ -0,0 +1,7 @@
+Tutorials
+=======================================
+
```

Comment:
Remove the first 3 lines

Reply:
This resulted in the tutorial.rst contents showing on Index.html. I put them back in.